paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -1505,6 +1505,17 @@
   kernel:
     func: seed
 
+- op : send_and_recv
+  args : (Tensor[] x, str message_name, str[] send_var_name, str[] recv_var_name, int trainer_id = 0, str mode="forward", str[] endpoints={"127.0.0.1:6164"}, str[] next_endpoints={"127.0.0.1:6164"}, str[] previous_endpoints={"127.0.0.1:6164"})
+  output : Tensor[](out)
```

```suggestion
  output : Tensor[](out){x.size()}
```
这里需要加上`{x.size()}`,代码生成脚本如果发现没有`x.size()`会使用None填充，导致编译错误